### PR TITLE
Reorganize tabs

### PR DESCRIPTION
**Pull request by jrmi**
for better UX

### CODE REVIEW

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 1]
```diff
@@ -178,55 +182,135 @@ const DataLayerForm = ({
...
+                      label="datalayer.form.type.label"
+                      validate={defaultRequired}
+                    />
+                    {filterType && (
+                    <BooleanInput
+                      source={getSource('filter_settings.fetchValues')}
+                      label="datalayer.form.filter.type.fetch.label"
+                      fullWidth
+                    />
+                    )}
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:267]:
```suggestion
                    {filterType && (
                      <BooleanInput
                        source={getSource('filter_settings.fetchValues')}
                        label="datalayer.form.filter.type.fetch.label"
                        fullWidth
                      />
                    )}
```

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 2]
```diff
@@ -178,55 +182,135 @@ const DataLayerForm = ({
...
+                    {(filterType && !filterFetch) && (
+                    <TextArrayInput
+                      source={getSource('filter_settings.values')}
+                      label={`datalayer.form.filter.type.values${filterType === 'many' ? '' : '_optional'}`}
+                      validate={filterType === 'many' ? defaultRequired : undefined}
+                      fullWidth
+                    />
+                    )}
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:275]:
```suggestion
                    {(filterType && !filterFetch) && (
                      <TextArrayInput
                        source={getSource('filter_settings.values')}
                        label={`datalayer.form.filter.type.values${filterType === 'many' ? '' : '_optional'}`}
                        validate={filterType === 'many' ? defaultRequired : undefined}
                        fullWidth
                      />
                    )}
```

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 3]
```diff
@@ -178,55 +182,135 @@ const DataLayerForm = ({
...
+                    {filterType === 'range' && (
+                    <SelectInput
+                      source={getSource('filter_settings.format')}
+                      label="datalayer.form.filter.type.range_format.label"
+                      choices={[
+                        { id: 'number', name: 'datalayer.form.filter.type.range_format.number' },
+                        { id: 'date', name: 'datalayer.form.filter.type.range_format.date' },
+                      ]}
+                    />
+                    )}
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:285]:
```suggestion
                    {filterType === 'range' && (
                      <SelectInput
                        source={getSource('filter_settings.format')}
                        label="datalayer.form.filter.type.range_format.label"
                        choices={[
                          { id: 'number', name: 'datalayer.form.filter.type.range_format.number' },
                          { id: 'date', name: 'datalayer.form.filter.type.range_format.date' },
                        ]}
                      />
                    )}
```

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 4]
```diff
@@ -236,74 +320,25 @@ const DataLayerForm = ({
...
-                {formData.table_enable && <BooleanInput source="shown" label="datalayer.form.show" />}
+
+                {formData.table_enable ? <BooleanInput source="shown" label="datalayer.form.table.show" /> : <></>}
                 <FormDataConsumer>
                   {({ scopedFormData = {}, getSource }) => (
                     scopedFormData.shown && <BooleanInput source={getSource('display')} label="Afficher ce champs par défaut" />
                   )}
                 </FormDataConsumer>
-                {formData.table_export_enable && <BooleanInput source="exportable" label="datalayer.form.exportable" />}
+                {formData.table_export_enable ? <BooleanInput source="exportable" label="datalayer.form.table.exportable" /> : <></>}
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:330]:
```suggestion
                {formData.table_export_enable && <BooleanInput source="exportable" label="datalayer.form.table.exportable" />}
```

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 5]
```diff
@@ -236,74 +320,25 @@ const DataLayerForm = ({
...
-                            ]}
-                          />
-                        )}
-                      </>
-                    );
-                  }}
-                </FormDataConsumer>
-                {formData.table_enable && <BooleanInput source="shown" label="datalayer.form.show" />}
+
+                {formData.table_enable ? <BooleanInput source="shown" label="datalayer.form.table.show" /> : <></>}
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:324]:
```suggestion
                {formData.table_enable && <BooleanInput source="shown" label="datalayer.form.table.show" />}
```

File: src/modules/RA/DataLayer/components/DataLayerForm.js [hunk 6]
```diff
@@ -129,7 +129,7 @@ const DataLayerForm = ({
 
       </LazyFormTab>
 
-      <LazyFormTab disabled={external} label="datalayer.form.style" path="style">
+      <LazyFormTab disabled={external} label="datalayer.form.styles.tab">
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerForm.js:132]: Why having remove `path="style"` prop ?

Comment (jrmi) [reply src/modules/RA/DataLayer/components/DataLayerForm.js]: I misunderstood the utility of this property.
Fixed

**Review by CFarcy — APPROVED** [review]

**Pull request merged**